src/components/BasicInfo.jsx

```diff
@@ -0,0 +1,66 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import StyledFieldset from './StyledFieldset';
+
+export default function BasicInfo({
+  firstName,
+  lastName,
+  pronouns,
+  setRegisterStep,
+  handleFormChange,
+}) {
+  const filledOut = firstName !== '' && lastName !== '' && pronouns !== '' && true;
+  return (
+    <StyledFieldset>
+      <legend>Basic Info</legend>
+      <p>Input basic info about yourself.</p>
+      <p className="warning">*Fields are required</p>
+      <label htmlFor="firstName">
+        First Name<span className="warning">*</span>
+        <input
+          id="firstName"
+          type="text"
+          name="firstName"
+          onChange={handleFormChange}
+          value={firstName}
+        />
+      </label>
+      <label htmlFor="lastName">
+        Last Name<span className="warning">*</span>
+        <input
+          id="lastName"
+          type="text"
+          name="lastName"
+          onChange={handleFormChange}
+          value={lastName}
+        />
+      </label>
+      <label htmlFor="pronouns">
+        Pronouns<span className="warning">*</span>
+        <input
+          id="pronouns"
+          type="text"
+          name="pronouns"
+          onChange={handleFormChange}
+          value={pronouns}
+        />
+      </label>
+      <nav>
+        <button
+          onClick={filledOut ? () => setRegisterStep(2) : null}
+          className={filledOut ? '' : 'inactive'}
```

Comment:
Can you disable the buttons using the disabled attribute instead?
A user using a screen reader won't pick up on the color changes.